packages/manager/modules/enterprise-cloud-database/src/service/details/settings/settings.routing.js

```diff
@@ -0,0 +1,16 @@
+export default /* @ngInject */($stateProvider) => {
+  $stateProvider.state('enterprise-cloud-database.service.details.settings', {
+    component: 'enterpriseCloudDatabaseServiceDetailsSettingsComponent',
+    resolve: {
+      maintenanceWindow: /* @ngInject */
+        ($q, clusterId, enterpriseCloudDatabaseService) => enterpriseCloudDatabaseService
+          .getMaintenanceWindow(clusterId)
+          .catch(error => ((error.status === 404) ? null : $q.reject(error))),
```

Comment:
Could it make sense to catch it in `enterpriseCloudDatabaseService` ? Is that a behaviour that we might always want

Reply:
This is done. Thanks for the suggestion :)